### PR TITLE
Add YUYV to supported video formats

### PR DESCRIPTION
**Pull request by RyanBoring**
#### Description
Add YUYV to supported video formats; used to be implicitly supported under YUY2. That meant the user had to know to ask for YUY2 decoding when using a YUYV device; this PR allows the user to specify YUYV instead.

### CODE REVIEW

**Comment by RyanBoring** [comment]
@bazile-clyde Would you be able to review this for me? I can't add you as a reviewer directly

**Comment by bazile-clyde** [comment]
> @bazile-clyde Would you be able to review this for me? I can't add you as a reviewer directly

@RyanBoring yep! Looking now.

File: pkg/frame/decode.go [hunk 1]
```diff
@@ -17,6 +17,8 @@ const (
 	FormatNV12 = "NV12"
 	// FormatYUY2 https://www.fourcc.org/pixel-format/yuv-yuy2/
 	FormatYUY2 = "YUY2"
+	// FormatYUYV https://www.fourcc.org/pixel-format/yuv-yuy2/
```
Comment (bazile-clyde) [inline pkg/frame/decode.go:20]: This link isn't working for me. I think it's gone stale. Maybe figure our if _fourcc.org_ has moved to another domain, and if so where. Or you could paste another authoritative link about the YUYV format.

Comment (RyanBoring) [reply pkg/frame/decode.go]: Wow yeah great catch, I just changed them in a new commit

**Review by bazile-clyde — APPROVED** [review]
Nice! LGTM!

File: pkg/frame/decode.go (hunk 1 above)
Comment (bazile-clyde) [reply pkg/frame/decode.go]: Thank you!

**Review by bazile-clyde — APPROVED** [review]

**Pull request merged**